src/initializers.jl

```diff
@@ -0,0 +1,50 @@
+# Initializer interface
```

Comment:
I know there's already an explanation in the general docs, but I think the initialiser interface could use some more explanation in doc strings as well somewhere. Especially with regards to how this is used in the end in the example scripts, which isn't immediately clear I think from the docs as is.